concepticondata/concepticon.tsv

```diff
@@ -2064,7 +2064,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 2063	CLOSE (EYES)	Basic actions and technology	Move ones eyelids in such a way that the eyes are no longer visible.	Action/Process	
 2064	SCATTER (SEEDS)	Basic actions and technology	To distribute seeds over a field.	Action/Process	
 2065	STEP (VERB)	Basic actions and technology	To move the foot in walking; to advance or recede by raising and moving one of the feet to another resting place, or by moving both feet in succession.	Action/Process	
-2066	PULL UP (RICE SEEDLINGS)	Basic actions and technology		Action/Process	
+2066	PULL UP (RICE SEEDLINGS)	Basic actions and technology	To remove the roots of a rice plant.	Action/Process	
```

Comment:
That sounds strange, isn't it rather something like "to pull a rice seedling out of the ground"?
I remember @MacyL and me talked about that concept before but I don't remember the details. Only thing mapped to it is 拔（秧）.

Reply:
I used the definition of [pull up](https://www.collinsdictionary.com/dictionary/english/pull-up) in the Collins dictionary as a basis.

Reply:
"to remove by the roots"? Ah, now I see how you mean your definition.
"To remove the roots of a rice plant." sounds to me like "To remove the roots from a rice plant, and doing whatever with the remainder.", but you mean "To remove a rice plant including the roots.".

Rice seedlings are nothing usually just discarded, though, so I would guess this is rather about harvesting them for food or replanting them where there is more space. @MacyL could you help?

Reply:
Suggestion: "To remove a rice plant including its roots (for harvesting)."

Reply:
Here is my thought : remove PULL UP (RICE SEEDLINGS), and move Allen-2007-500-296 to PULL UP WEEDS.
Reason : 
1. Only one concept, 拔（秧）(Allen-2007-500-296) map to this concepticon concept.
2. The form "ma²¹" (Allenbai, Jianchuan language variety) is equal to the form "ma21" in (Wangbai, doculact DS). And Wang-2004-471-311 pull up (weeds) maps to PULL UP WEEDS (along with many others)
3. In many concept list with Chinese glosses, the object in the brackets is an example. So we should focus on defining the action 拔  "pull up". 
4. When harvesting rice (稻米）, Chinese don't pull up the rice plants （稻株）, the farmers cut the plants instead of pull up. The method varies between countries.
5. 拔秧 (not brackets are used here), if there is a concept like this,  it means pull up rice seedlings including the roots (as Annika suggested) . But harvesting is not the main reason, it's for spacing rice seedlings, so that they have room to grow (and subsequently, maximising the quantity of rice).

Reply:
Thanks! Then I'd suggest we remap Allen to PULL UP WEEDS.
We could either retire this concept here, or we leave it open for future mappings and change the definition to something like "To pull out rice seedlings from the soil including their roots in order to separate seedlings growing too closely.".